pkg/kubelet/userns/userns_manager.go

```diff
@@ -229,7 +253,10 @@ func (m *UsernsManager) record(pod types.UID, from, length uint32) (err error) {
 	if found && prevFrom != from {
 		return fmt.Errorf("different user namespace range already used by pod %q", pod)
 	}
-	index := int(from / userNsLength)
+	index := int(from/userNsLength) - m.off
+	if index >= m.len {
```

Comment:
Grrr, this shouldn't happen if the API for the bitmap was better. We shouldn't care about this, it should just fail to allocate one if there is no more room or something :-/

Reply:
Same brain-fart as before, no issue here